tests/Extensions/Directory.Build.props

```diff
@@ -1,19 +1,14 @@
 <Project>
   <Import Project="$([MSBuild]::GetPathOfFileAbove('$(_DirectoryBuildPropsFile)', '$(MSBuildThisFileDirectory)../'))" />
   <PropertyGroup>
-    <NetFameworkVersion>net472</NetFameworkVersion>
-    <NetCoreVersion>netcoreapp3.1</NetCoreVersion>
-    <TargetFrameworksValue>$(NetFameworkVersion);$(NetCoreVersion)</TargetFrameworksValue>
     <LangVersion>latest</LangVersion>
     <Nullable>enable</Nullable>
-    <IsPackable>false</IsPackable>
-    <IsTestProject>true</IsTestProject>
```

Comment:
Do we definitely want to remove `IsTestProject`? It seems like something that would be beneficial, but I can't find much useful information about it on the web

Reply:
It's defined in the `tests/Directory.Build.props` that is also applied here